### PR TITLE
feat: highlight config and drop card treatment

### PR DESCRIPTION
**Pull request by lauchness**
## Description
Highlight config and changes from #351

### CODE REVIEW

**Review by AshishYelekar — APPROVED** [review]
@wilsoncusack this lgtm pending approval on the preview from the team / partners

**Comment by cb-heimdall** [comment]
Review Error for AshishYelekar @ 2023-08-21 21:36:46 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by wilsoncusack — APPROVED** [review]

**Pull request merged**
